src/openMVG/sfm/pipelines/sequential/sequential_SfM.cpp

```diff
@@ -389,9 +389,7 @@ bool SequentialSfMReconstructionEngine::ChooseInitialPair(Pair & initialPairInde
 
     if (_sfm_data.GetIntrinsics().empty() || valid_views.empty())
     {
-      std::cerr
-        << "There is no defined intrinsic data in order to compute an essential matrix for the initial pair."
-       );
+      std::cerr << "There is no defined intrinsic data in order to compute an essential matrix for the initial pair.";
```

Comment:
`OPENMVG_CERR("There is no defined intrinsic data in order to compute an essential matrix for the initial pair.");`